### PR TITLE
Make the "sign in or sign up" clickable in banner

### PR DESCRIPTION
**Pull request by yehudab**
Fixes: #5054

### CODE REVIEW

File: locales/en-US/server.ftl [hunk 1]
```diff
@@ -15,7 +15,7 @@ buttonSignIn =
     .title = Sign In
 screenshotsLogo =
     .title = Screenshots Home
-bannerMessage = Sign in or sign up to access your shots across devices and save your favorites forever.
+bannerMessage = <a>Sign in or sign up</a> to access your shots across devices and save your favorites forever.
```
Comment (flodolo) [inline locales/en-US/server.ftl]: Sorry, this [kind of changes](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings) require a new string ID

**Comment by 6a68** [comment]
@yehudab Thanks for the PR. This bug hasn't been triaged yet (it's not in a milestone), so I'm not sure this change will actually land; we should get a product decision from John later this week.

In general, good-first-issue labels or stuff in the Stretch milestone is a safe bet if you're looking for ways to contribute :+1: (and I'll update our docs to make that clearer, sorry)

**Comment by yehudab** [comment]
Thanks for the clarification. No worries if it doesn’t land soon. It was also a good practice for me to understand the l10n process and best practices.

File: server/src/ad-banner.js [hunk 2]
```diff
@@ -25,9 +29,12 @@ exports.AdBanner = class AdBanner extends React.Component {
           </p>
         </Localized>;
     } else if (this.props.isOwner && !this.props.hasFxa) {
```
Comment (punamdahiya) [inline server/src/ad-banner.js]: With #4959 fix, condition here has changed to 

`} else if (!this.props.hasFxa) {`

**Comment by yehudab** [comment]
@punamdahiya PR is now rebased and squashed

**Comment by yehudab** [comment]
Documentation updated.

**Comment by punamdahiya** [comment]
> Documentation updated.

Thank you! @yehudab

**Pull request merged**